src/tests/topp/IDMerger_6_output.idXML

```diff
@@ -0,0 +1,514 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="https://www.openms.de/xml-stylesheet/IdXML.xsl" ?>
+<IdXML version="1.5" xsi:noNamespaceSchemaLocation="https://www.openms.de/xml-schema/IdXML_1_5.xsd" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
```

Comment:
nice that you also provide the human readable one.
is this also tested against e.g. the output from the old format? e.g., can we trust this output?

Reply:
It's not currently tested with and without conversion, but I'm pretty sure I've checked the output file for completeness.